src/components/Components/component-actions.tsx

```diff
@@ -15,6 +15,7 @@ export const useComponentActions = (component: ComponentKind, name: string): Act
   const applicationName = component?.spec.application;
   const [canPatchComponent] = useAccessReviewForModel(ComponentModel, 'patch');
   const [canDeleteComponent] = useAccessReviewForModel(ComponentModel, 'delete');
+  const [canManageLinkedSecrets] = useAccessReviewForModel(SecretModel, 'get');
```

Comment:
I'm just not sure if the `verb` here should be `get` or `list` (or something else 😄)

Reply:
We don't really need a check for a `Get` or `list`? 
Linked secrets are added to the service account `appstudio-pipeline`. Hence, if we need a check it shouldn't it be for a patch-request to ServiceAccount Model??

Reply:
We don't really need a check for a `Get` or `list`?
Linked secrets are added to the service account `appstudio-pipeline`. Hence, if we need a check it shouldn't it be for a `patch-request` to `ServiceAccount` Model??

Reply:
With the new modal, the shared appstudio-pipeline sa won't exist.
see the design picture:
![Screenshot 2025-04-29 at 15 10 26](https://github.com/user-attachments/assets/ca5c17e4-88cb-4822-bd12-ea824f3099b3)

Reply:
I did some tests here, only admin users of one namespace are able to view/create/update/delete one secrets. Maintainers & contributors miss all permission for secrets. In this content, checking one action permission of secrets = checking all permissions. To this content, 'get' works well functionally.

But i agree with Abhi.  there is no other permission check for the page and users are able to manage the secret. So let us change it to 'patch'. :-)

Reply:
updated :)